controllers/users.js

```diff
@@ -39,21 +52,77 @@ function modifyResponse(submitted, stored) {
   return stored;
 }
 
-function updatePreferences(uuid, questions_answered, callback=(res)=>{}) {
-  User.findOne({ "uuid": uuid }, 
-    function(err, result) {
+function updateStance(uuid, questions_answered, callback=(res)=>{}) {
+  findById(uuid, (result) => {
       User.updateOne({ "uuid": uuid }, { 
         $set: {
           questions_answered: modifyResponse(questions_answered, 
                                              result.questions_answered)
         },
-      }, function(err, result) {
+      }, (err, result) => {
         callback(result);
       })
  });
 }
 
+function incrementBadgeCount(currentBadges, badgeName){
+  console.log('increment Badge Count Called');
+  console.log(currentBadges);
+  console.log(badgeName);
+  console.log('arguments above');
+  var idx = currentBadges.map(function(d){ return d.badge; }).indexOf(badgeName);
```

Comment:
please put callbacks on a new line 

you could also do `d => d.badge` to be a little more consise

Reply:
also what does `idx` mean